saleor/graphql/account/mutations/deprecated_account.py

```diff
@@ -0,0 +1,172 @@
+            required=False,
+            description=(
+                "A type of address. If provided, the new address will be "
+                "automatically assigned as the customer's default address "
+                "of that type."
+            ),
+        )
+
+    class Meta:
+        description = (
+            "DEPRECATED: Use AccountAddressCreate instead."
+            "Create a new address for the customer."
+        )
+        model = models.Address
+        exclude = ["user_addresses"]
+
+    @classmethod
+    def check_permissions(cls, user):
+        return user.is_authenticated
+
+    @classmethod
+    def perform_mutation(cls, root, info, **data):
+        success_response = super().perform_mutation(root, info, **data)
+        address_type = data.get("type", None)
+        if address_type:
+            user = info.context.user
+            instance = success_response.address
+            utils.change_user_default_address(user, instance, address_type)
+            success_response.user = user
+        return success_response
+
+    @classmethod
+    def save(cls, info, instance, cleaned_input):
+        super().save(info, instance, cleaned_input)
+        user = info.context.user
+        instance.user_addresses.add(user)
+
+
+class CustomerSetDefaultAddress(BaseMutation):
+    user = graphene.Field(User, description="An updated user instance.")
+
+    class Arguments:
+        id = graphene.ID(
+            required=True, description="ID of the address to set as default."
+        )
+        type = AddressTypeEnum(required=True, description="The type of address.")
+
+    class Meta:
+        description = (
+            "DEPRECATED: Use AccountSetDefaultAddress instead."
+            "Sets a default address for the authenticated user."
+        )
+
+    @classmethod
+    def check_permissions(cls, user):
+        return user.is_authenticated
+
+    @classmethod
+    def perform_mutation(cls, _root, info, **data):
+        address = cls.get_node_or_error(info, data.get("id"), Address)
+        user = info.context.user
+
+        if address not in user.addresses.all():
+            raise ValidationError({"id": "The address doesn't belong to that user."})
+
+        if data.get("type") == AddressTypeEnum.BILLING.value:
+            address_type = AddressType.BILLING
+        else:
+            address_type = AddressType.SHIPPING
+
+        utils.change_user_default_address(user, address, address_type)
+        return cls(user=user)
+
+
+class CustomerPasswordResetInput(graphene.InputObjectType):
+    email = graphene.String(
+        required=True,
+        description=("Email of the user that will be used for password recovery."),
+    )
+
+
+class CustomerPasswordReset(BaseMutation):
+    class Arguments:
+        input = CustomerPasswordResetInput(
+            description="Fields required to reset customer's password", required=True
+        )
+
+    class Meta:
+        description = "Resets the customer's password."
```

Comment:
If this mutation is deprecated we should say in the description which one to use instead.